### PR TITLE
spelling fixes and a better warning for not using DXVK

### PR DESCRIPTION
**Pull request by dribbleondo**
There were a load of spelling errors, and weird "heres whats going wrong, heres how proton fixed it". Generally, if people want to look at an FAQ for answers, you should probably assure them that a later version of proton has fixed their issue first, then the troubleshooting steps next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,61 +41,62 @@ With this FAQ, we want to cover the most important questions related to the usag
...
 
-## Why do my games crash on start, run very slow or have rendering issues?
+## Why do my games crash on startup, run very slowly or have rendering issues?
 
-#### Be sure that your system is up-to-date and that you are using the latest drivers available your graphics card. You will find detailed informations about how to update your graphic drivers in our [Wiki](https://github.com/NoXPhasma/protondb_faq/wiki/Graphics-driver-installation)
+#### Be sure that your system is up-to-date and that you are using the latest drivers available for your graphics card. Generally, for AMD and Intel, you should have be using a recent version of Mesa (19+) if you're on AMD or Intel, and Nvidia users should insteall the proprietary drivers from Nvidia's site:  You will find more detailed information about how to update your graphic drivers on our [Wiki](https://github.com/NoXPhasma/protondb_faq/wiki/Graphics-driver-installation)
```
Comment (ghost) [inline README.md]: Its not recommended to install nvidia from nvidia's site.

**Review by ghost — CHANGES_REQUESTED** [review]
Its not recommended to install nvidia over nvidia's site. Otherwise, it looks good.

File: README.md (hunk 1 above)
Comment (dribbleondo) [reply README.md]: Done.

**Review by ghost — CHANGES_REQUESTED** [review]
Its better that "’Generally speaking, for AMD and Intel users, you should be using a recent version of Mesa (19+ is a good idea), and Nvidia users should install version 415 of the proprietary drivers’" should be written in the wiki instead of here.

**Comment by dribbleondo** [comment]
Okay, I'll remove that.

**Comment by ghost** [comment]
LGTM, Thanks for your contribution!

**Pull request merged**